webapp/src/utils/config.spec.ts

```diff
@@ -38,10 +40,50 @@ describe('LyraConfig', () => {
       });
 
       const config = await LyraConfig.readFromDir('/path/to/repo');
-      expect(config.messagesPath).toEqual('/path/to/repo/subproject/locale');
-      expect(config.translationsPath).toEqual(
+      expect(config.projects[0].messagesPath).toEqual(
+        '/path/to/repo/subproject/locale'
+      );
+      expect(config.projects[0].translationsPath).toEqual(
         '/path/to/repo/subproject/locale'
       );
     });
+
+    it('rad more than one projects', async () => {
```

Comment:
I think this should be called "reads more than one project".

Reply:
fixed